test/verify/machineslib.py

```diff
@@ -771,6 +794,16 @@ def verify_disk_added(self):
             expected_target='vdh',
         ).execute()
 
+        if "debian" not in m.image and "ubuntu" not in m.image:
+            # ISCSI driver does not support virStorageVolCreate API
```

Comment:
This comment is confusing -- does that apply to libvirt in Debian/Ubuntu? or to some particular properties below?

Reply:
Oh, it was supposed to be explanation in why I used use_existing_volume='True' in the test bellow, and didn't test both scenarios (create new volume / use existing volume) as for example in nfs case.
libvirt virStorageVolCreateAPI is not capable to create volumes on iscsi pools, so such test wouldn't make sense.

Reply:
And of course it's not related to only debian/ubuntu, it's the same for all distros.